src/Vimeo.js

```diff
@@ -101,12 +91,12 @@ THE SOFTWARE. */
       }
 
       if(this.options_.poster == "") {
-        $.getJSON(this.baseApiUrl + this.videoId + '.json?callback=?', {format: "json"}, (function(_this){
-          return function(data) {
-            // Set the low resolution first
-            _this.setPoster(data[0].thumbnail_large);
-          };
-        })(this));
+        // $.getJSON(this.baseApiUrl + this.videoId + '.json?callback=?', {format: "json"}, (function(_this){
```

Comment:
http://www.w3schools.com/js/js_json_http.asp
https://mathiasbynens.be/notes/xhr-responsetype-json